apis/v1/events/schema.yaml

```
@@ -331,6 +331,70 @@ paths:
           $ref: 'http://localhost:3320/commons/components#/components/responses/404'
         '500':
           $ref: 'http://localhost:3320/commons/components#/components/responses/500'
+  "/api/processed-messages/{idempotencyKey}":
+    post:
+      tags:
+      - Mensagens
+      summary: Reenvie uma mensagem processada a partir do seu id de idempotência.
```

O summary acaba sendo sempre o título da página na nossa doc ;) 

```suggestion
      summary: Reprocessamento de mensagens
```